examples/06_Combining_Models.ipynb

```diff
@@ -6,6 +6,250 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Composite Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A CompositeModel is a PrognosticsModel that is composed of multiple PrognosticsModels. This is a tool for modeling system-of-systems. i.e., interconnected systems, where the behavior and state of one system effects the state of another system. The composite prognostics models are connected using defined connections between the output or state of one model, and the input of another model. The resulting CompositeModel behaves as a single model.\n",
+    "\n",
+    "To illustrate this, we will create a compositemodel of an aircraft's electric powertrain, combining the DCMotor, ESC, and PropellerLoad models.\n",
+    "\n",
+    "First we will import the used models, and the CompositeModel class"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.models import DCMotor, ESC, PropellerLoad\n",
+    "from progpy import CompositeModel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will initiate objects of the individual models that will later create the composite powertrain model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m_motor = DCMotor()\n",
+    "m_esc = ESC()\n",
+    "m_load = PropellerLoad()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we have to define the connections between the systems. Let's first define the connections from the DCMotor to the propeller load."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('motor states: ', m_motor.states)\n",
+    "print('load inputs: ', m_load.inputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each of the states and inputs are described in the model documentation at [DC Motor Docs](https://nasa.github.io/progpy/api_ref/prog_models/IncludedModels.html#dc-motor) and [Propeller Docs](https://nasa.github.io/progpy/api_ref/prog_models/IncludedModels.html#propellerload)\n",
+    "\n",
+    "From reading the documentation we understand that the propeller's velocity is from the motor, so we can define the first connection:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "connections = [\n",
+    "    ('DCMotor.v_rot', 'PropellerLoad.v_rot')\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The connection above telling the composite model to feed the DCMotor's v_rot into the propeller load's input v_rot.\n",
```

Comment:
```suggestion
    "The connection above tells the composite model to feed the DCMotor's v_rot into the PropellerLoad's input v_rot.\n",
```
for consistency, so it's clear PropellerLoad is a model

Reply:
Somewhere in here, I think we should explicitly state that the connections are written in a way that the first value is the input for the second value. Just to make the notation and connections very clear. What do you think? I'm not sure how to word it